app/views/govuk_publishing_components/components/_contents_list.html.erb

```diff
@@ -41,7 +53,8 @@
               track_label: contents_item[:href],
               track_options: {
                 dimension29: contents_item[:text]
-              }
+              },
+              ga4_link: {"index": {"index_link": position}, "index_total": contents.length}.to_json
```

Comment:
Does this also need some kind of `if ga4_tracking` check?

Reply:
Good spot, will add one.